src/test/java/io/vertx/tests/cache/spi/CacheSpiBase.java

```diff
@@ -0,0 +1,111 @@
+package io.vertx.tests.cache.spi;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.httpproxy.cache.CacheOptions;
+import io.vertx.httpproxy.spi.cache.Cache;
+import io.vertx.httpproxy.spi.cache.Resource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.time.Instant;
+
+@RunWith(VertxUnitRunner.class)
+public abstract class CacheSpiBase {
```

Comment:
We usually suffix such classes with `TestBase`:

```suggestion
public abstract class CacheSpiTestBase {
```